src/Features/Core/Portable/ImplementInterface/AbstractImplementInterfaceService.CodeAction.cs

```diff
@@ -41,12 +41,14 @@ internal partial class ImplementInterfaceCodeAction : CodeAction
                 State state,
                 bool explicitly,
                 bool abstractly,
+                bool onlyUnimplemented,
```

Comment:
based on the name of the feature, and how this value is used throughout this pr, i think this would make more sense as `onlyRemaining`.   then you can have `explicit: true, onlyRemaining: false` (for all methods) and `explicitTrue: onlyRemaining: true` for the remaining methods.